test/acceptance/behave/components/ngsiv2/entities/create_entities.feature

```diff
@@ -0,0 +1,1899 @@
+      | greater than max length allowed                |
+      | greater than max length allowed and ten levels |
+
+  @service_path_error @BUG_1092
+  Scenario:  try to create entities in NGSI v2 with several wrong service paths headers
+    Given  a definition of headers
+      | parameter          | value                                |
+      | Fiware-Service     | test_service_path_error              |
+      | Fiware-ServicePath | max length allowed and eleven levels |
+      | Content-Type       | application/json                     |
+    When create "1" entities with "1" attributes
+      | parameter        | value       |
+      | entities_type    | room        |
+      | entities_id      | room2       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    Then verify that receive several "Bad Request" http code
+    And verify several error responses
+      | parameter   | value                              |
+      | error       | BadRequest                         |
+      | description | too many components in ServicePath |
+    And verify that entities are not stored in mongo
+
+  # ---------- entities and attributes number --------------------------------
+  @entities_number
+  Scenario Outline:  create several entities in NGSI v2
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_entities    |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    When create "<entities>" entities with "1" attributes
+      | parameter        | value       |
+      | entities_type    | room        |
+      | entities_id      | room2       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    Then verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+    Examples:
+      | entities |
+      | 1        |
+      | 10       |
+      | 50       |
+      | 100      |
+
+  @attributes_number
+  Scenario Outline:  create entities in NGSI v2 with several attributes
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_attributes  |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    When create "1" entities with "<attributes>" attributes
+      | parameter        | value       |
+      | entities_type    | room        |
+      | entities_id      | room2       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    Then verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+    Examples:
+      | attributes |
+      | 1          |
+      | 10         |
+      | 50         |
+      | 100        |
+
+  @attributes_number_error
+  Scenario:  try to create entities in NGSI v2 without any attributes
```

Comment:
Only a comment...In this case the entity is created without attributes successfully